task6/task6.ipynb

```diff
@@ -228,8 +266,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For [6d](#6d) with the expected lifetime method we see higher CLV's for both constant and changing retention rates in comparison to using an infinite time horizon and the survival probabilities for each month. In 6d there is no discounting and it uses averages of retention rates either way. Meanwhile for [6c](#6c) we only use the average constant retention rate for cohort 0 and not the average across all cohorts. The average retention rate for cohort 0 is lower than the average of all cohorts. In addition to discounting the cash flow. The biggest difference is however accounting for cohort 0 and not averaging across all cohorts, since cohort 0 was really unprofitable compared to the other cohorts for last year. If we would have used the average retention rate across all cohorts, we would get almost the same expected CLV. In summary cohort 0, which is the month we are in now, was a much less profitable group of acquired users last year then the other cohorts, and we expect a lower CLV of $\\$12.69$ for this cohort."
+    "For [6d](#6d) with the expected lifetime method we see higher CLV's for both constant and changing retention rates in comparison to using an infinite time horizon and the survival probabilities for each month. For [6c](#6c) we only use the average constant retention rate for cohort 0 and not the average across all cohorts. The average retention rate for cohort 0 is lower than the average of all cohorts. In addition to discounting the cash flow. The biggest difference is however accounting for cohort 0 and not averaging across all cohorts, since cohort 0 was really unprofitable compared to the other cohorts for last year. If we would have used the average retention rate across all cohorts, we would get almost the same expected CLV. In summary cohort 0, which is the month we are in now, was a much less profitable group of acquired users last year then the other cohorts, and we expect a lower CLV of $\\$12.69$ for this cohort."
```

Comment:
Not sure if the value mentioned here is up do date

Reply:
It's the same as in 6c